# HG changeset patch
# User Rik <rik@octave.org>
# Date 1380074434 25200
#      Tue Sep 24 19:00:34 2013 -0700
# Node ID 6edc451629a6bcb7d7a2bb52e30af57f0feb6a85
# Parent  9849075a5da6edbfe28a432cbf84e6b8bedc3835
__contour__.m: Ignore dummy input in all listener functions.

* scripts/plot/private/__contour__.m: Ignore second input for update_text
callback function.

diff --git a/scripts/plot/private/__contour__.m b/scripts/plot/private/__contour__.m
--- a/scripts/plot/private/__contour__.m
+++ b/scripts/plot/private/__contour__.m
@@ -465,23 +465,23 @@ function update_data (h, ~, prop = "")
     else
       set (h, "levellist", lev);
       z = get (h, "zdata");
       lvlstep = (max (z(:)) - min (z(:))) / 10;
       set (h, "levelstep", lvlstep);
     endif
 
     add_patch_children (h);
-    update_text (h, d);
+    update_text (h);
     recursive = false;
   endif
 
 endfunction
 
-function update_text (h, d)
+function update_text (h, ~)
   persistent recursive = false;
 
   if (! recursive)
     recursive = true;
 
     delete (findobj (h, "type", "text"));
 
     if (strcmp (get (h, "textlistmode"), "manual"))
